src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -20,7 +20,9 @@ public class ArmSubsystem extends SubsystemBase {
 
   public final CANSparkMax m_winch = new CANSparkMax(ArmConstants.kWinchPort, CANSparkMaxLowLevel.MotorType.kBrushless); // Assume Brushless, unknown currently
```

Comment:
```suggestion
  public final CANSparkMax m_winch = new CANSparkMax(ArmConstants.kWinchPort, CANSparkMaxLowLevel.MotorType.kBrushed);
```
According to Philippe, this should be Brushed